helper-cli/src/main/kotlin/commands/ExportLicenseFindingCurationsCommand.kt

```diff
@@ -75,11 +75,8 @@ internal class ExportLicenseFindingCurationsCommand : CliktCommand(
             nestedRepositories = ortResult.repository.nestedRepositories
         )
 
-        val globalLicenseFindingCurations = if (licenseFindingCurationsFile.isFile) {
-            licenseFindingCurationsFile.readValue<RepositoryLicenseFindingCurations>()
-        } else {
-            mapOf()
-        }
+        val globalLicenseFindingCurations = licenseFindingCurationsFile.takeIf { it.isFile }
+            ?.readValue<RepositoryLicenseFindingCurations>().orEmpty()
```

Comment:
Shouldn't this have gone to `helper-cli/src/main/kotlin/commands/packageconfig/ExportLicenseFindingCurationsCommand.kt` instead?

Reply:
Argh right, good catch!

Reply:
Ok, you now also changed the other location, but this change it still there...